package/rmt-server.spec

```diff
@@ -224,9 +217,9 @@ getent passwd %{rmt_user} >/dev/null || \
 
 %post
 %service_add_post rmt-server.target rmt-server.service rmt-server-migration.service rmt-server-mirror.service rmt-server-sync.service
-cd /usr/share/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails secrets:setup >/dev/null
-cd /usr/share/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails runner -e production "Rails::Secrets.write({'production' => {'secret_key_base' => SecureRandom.hex(64)}}.to_yaml)"
-echo "The installation completed. Please run yast2-rmt to configure the package properly" >> /dev/stdout
+cd %{_datadir}/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails secrets:setup >/dev/null
+cd %{_datadir}/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails runner -e production "Rails::Secrets.write({'production' => {'secret_key_base' => SecureRandom.hex(64)}}.to_yaml)" >/dev/null
+echo "Please run the YaST RMT module (or yast2-rmt from the command line) to complete the configuration of your rmt-server" >> /dev/stdout
```

Comment:
I believe the command is `yast2 rmt` (no dash) but maybe `yast2-rmt` also works? :man_shrugging:

Reply:
also, I would say "...of your RMT server", or even just "of your RMT"

"RMT server" reads like "repository mirroring tool server" which is funky haha

Reply:
message updated